adapters/repos/db/lsmkv/segment_group.go

```diff
@@ -94,19 +108,19 @@ func newSegmentGroup(dir string, logger logrus.FieldLogger,
 		// If yes, we must assume that the flush never finished, as otherwise the
 		// WAL would have been lsmkv.Deleted. Thus we must remove it.
 		potentialWALFileName := strings.TrimSuffix(entry.Name(), ".db") + ".wal"
```

Comment:
We really shouldn't be hard coding file extensions

Reply:
that was not changed in this PR